bin/updateVersion.mjs

```diff
@@ -0,0 +1,120 @@
+#! /usr/bin/env node
+import semver from 'semver'
```

Comment:
semver should probably be added as a dependency, to ensure a predictable version is available to us